mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua

```diff
@@ -0,0 +1,496 @@
+			Trigger.RemoveFootprintTrigger(id)
+			foot9Triggered = true
+
+			local trig9camera = Actor.Create("camera", true, { Owner = allies, Location = BridgeCam.Location})
+			Trigger.AfterDelay(DateTime.Seconds(10), function()
+				trig9camera.Destroy()
+			end)
+
+			Utils.Do(BridgeMammoths, function(actor)
+				actor.AttackMove(MammysGo.Location)
+			end)
+		end
+	end)
+
+	local foot10Triggered
+	Trigger.OnEnteredFootprint(FootprintTrigger10, function(actor, id)
+		if actor.Owner == allies and not foot10Triggered then
+			Trigger.RemoveFootprintTrigger(id)
+			foot10Triggered = true
+
+			local trig10camera = Actor.Create("camera", true, { Owner = allies, Location = TruckCam.Location})
+			Trigger.AfterDelay(DateTime.Seconds(10), function()
+				trig10camera.Destroy()
+			end)
+
+			Media.PlaySpeechNotification(allies, "SignalFlareNorth")
+			Actor.Create("camera", true, { Owner = allies, Location = ExtractionPoint.Location})
+			Actor.Create("flare", true, { Owner = allies, Location = FlareUp.Location})
+			Actor.Create("flare", true, { Owner = allies, Location = FlareDown.Location})
+			Actor.Create("flare", true, { Owner = allies, Location = FlareLeft.Location})
+			Actor.Create("flare", true, { Owner = allies, Location = FlareRight.Location})
+			SendExtractionHelicopter()
+
+			HealCrateTruck.Move(TruckGo.Location)
+		end
+	end)
+
+	local foot11Triggered
+	Trigger.OnEnteredFootprint(FootprintTrigger11, function(actor, id)
+		if actor.Owner == allies and not foot11Triggered then
+			Trigger.RemoveFootprintTrigger(id)
+			foot11Triggered = true
+
+			local trig11camera = Actor.Create("camera", true, { Owner = allies, Location = SovBaseCam.Location})
+			Trigger.AfterDelay(DateTime.Seconds(10), function()
+				trig11camera.Destroy()
+			end)
+
+			local reinforcement = lstReinforcements.second
+			Media.PlaySpeechNotification(allies, "ReinforcementsArrived")
+			Reinforcements.ReinforceWithTransport(allies, "lst.reinforcement", reinforcement.actors, reinforcement.entryPath, reinforcement.exitPath)
+		end
+	end)
+
+	local foot12Triggered
+	Trigger.OnEnteredFootprint(FootprintTrigger12, function(actor, id)
+		if (actor.Type == "gnrl" or actor.Type == "gnrl.noautotarget") and not foot12Triggered then
+			Trigger.RemoveFootprintTrigger(id)
+			foot12Triggered = true
+
+			Media.PlaySoundNotification(allies, "AlertBleep")
+			Media.DisplayMessage("Stalin will pay for what he has done today! \nI will bury him with my own hands!", "Stavros")
+		end
+	end)
+end
+
+ChurchAttack = function()
+	if not ChurchDamaged then
+		Reinforcements.Reinforce(greece, CivilianSquad, { ChurchSpawn.Location, GrenEntry.Location }, 0)
+	end
+	ChurchDamaged = true
+end
+
+SendMig = function(waypoints)
+	local MigEntryPath = { waypoints[1].Location, waypoints[2].Location }
+	local Mig = Reinforcements.Reinforce(ussr, { "mig" }, MigEntryPath)
+	if not BridgeBarrel1.IsDead then
+		Utils.Do(Mig, function(mig)
+			mig.Attack(BridgeBarrel1)
```

Comment:
Try a forced attack:

```suggestion
			mig.Attack(BridgeBarrel1, true, true)
```

Reply:
Got the same result.

Reply:
Try setting a waypoint so it flies towards it and do the `Attack` when it sees the target.

Reply:
Got it to work by giving ussr a camera there